.github/workflows/lint.yml

```diff
@@ -16,8 +16,16 @@ jobs:
       - name: Install Foundry
         uses: foundry-rs/foundry-toolchain@v1
         with:
-          version: v1.2.3
+          version: v1.3.5
 
-      - name: Lint
+      - name: fmt
         run: |
           forge fmt --check
+
+      - name: build
+        run: |
+          forge build
+
+      - name: Lint
+        run: |
+          ! (forge lint 2>&1 | grep "")
```

Comment:
Seems we can use `--deny=warnings` starting on v1.4.0 but that version is not yet released.